mode/examples/Topics/Motion/CubesWithinCube/CubesWithinCube.pyde

```diff
@@ -6,22 +6,21 @@ Collision detection against all outer cube's surfaces.
 """
 from cube import Cube
 
-halfWidth = None
-halfHeight = None
 
 Bounds = 300
 # 20 little internal cubes
 cubes = []
-for i in range(20):
```

Comment:
I hear you. It seems to me like something which belongs in (the aptly named) `setup`.